docs/ops/signals/IRDFT_9.md

```diff
@@ -0,0 +1,210 @@
+# Inverse Discrete complex-to-real Fourier Transformation (IRDFT) {#openvino_docs_ops_signals_IRDFT_9}
+
+**Versioned name**: *IRDFT-9*
+
+**Category**: *Signal processing*
+
+**Short description**: *IRDFT* operation performs the inverse complex-to-real discrete Fourier transformation of input tensor by specified dimensions.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+*   **1**: `data` - Input tensor of type *T* with data for the IRDFT transformation. Type of elements is any supported floating-point type. The last dimension of the input tensor must be equal to 2, that is the input tensor shape must have the form `[D_0, D_1, ..., D_{N-1}, 2]`, representing the real and imaginary components of complex numbers in `[:, ..., :, 0]` and in `[:, ..., :, 1]` correspondingly. **Required.**
+*   **2**: **2**: `axes` - 1D tensor of type *T_IND* specifying dimension indices where IRDFT is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, for example, `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`, `[-3, 0, -2]`. These indices should be integers from `-(r - 1)` to `(r - 2)` inclusively, where `r = rank(data)`. A negative axis `a` is interpreted as an axis `r - 1 + a`. Other dimensions do not change. The order of elements in `axes` attribute matters, and is mapped directly to elements in the third input `signal_size`. **Required.**
+*   **NOTE**: The following constraint must be satisfied: `rank(data) >= len(axes) + 1 and input_shape[-1] == 2 and (rank(data) - 1) not in axes and (-1) not in axes`.
```

Comment:
Better to move input_shape[-1] == 2 into the first input description and rename like data_shape.

Reply:
Done.